scripts/show-features.py

```diff
@@ -100,7 +100,7 @@ def main(argv=None):
             extractor = capa.features.freeze.load(f.read())
     else:
         try:
-            extractor = capa.main.get_extractor(args.sample, args.format, capa.main.BACKEND_VIV)
+            extractor = capa.main.get_extractor(args.sample, args.format, capa.main.BACKEND_VIV, sigpaths=[])
```

Comment:
similar concern here, although we'll see in practice if this is really an issue, could be helpful to see an indication on "you're looking at a library function" which I know has happened in the past for proposed rules

Reply:
hmmm.... yeah i wonder what the best thing to do here is. maybe the presence of the `API` feature at a function will indicate a given address is a library function?

Reply:
hm, but features aren't even extracted from library functions. i'd suggest that we do the same from show-features

Reply:
bd92933

Reply:
looks good to me!